addons/sale_pdf_quote_builder/models/sale_order_line.py

```diff
@@ -25,6 +25,9 @@ class SaleOrderLine(models.Model):
 
     @api.depends('product_id', 'product_template_id')
     def _compute_available_product_document_ids(self):
+        if not self.env.user.has_group('sales_team.group_sale_salesman'):
+            self.available_product_document_ids = False
+            return
```

Comment:
Hm, I think this compute should always work as expected, in order to not suddenly have some PDF not created correctly because suddenly a selected PDF isn't in the available list any more.

BTW, I think this restriction on the salesman for the field `attached_on_sale` might have been a mistake, as it was added when refactoring mrp. I don't see why someone that can see the document can't know where it should be used for. But that might be a question for a PO.

Reply:
I guess user with no sales access should not able to call `get_update_included_pdf_params`  as it is giving access error when trying to fetch data from SOL because user does not have SOL access. Should I remove check :thinking: 
(It was working in my case because I was setting that field to false when user does not have access)

Reply:
Or maybe hide the quote builder page, that way, the compute won't get triggered. But might be interesting to add a sudo to ensure the available documents are always the same and don't depend on who is opening the order

Reply:
I guess they can't  as related menu already have sale user group so internal user can't access this this view!  should be good now :see_no_evil: